check/check_test.go

```diff
@@ -192,3 +193,70 @@ func TestGetFirstValidSubCheck(t *testing.T) {
 		}
 	}
 }
+func TestTextToCommand(t *testing.T) {
+	type TestCase struct {
+		auditCommand          string
+		expectedSlicedCommand []string
+	}
+
+	// For each test case, we want get the different commands separated by | and exclude the \\|
+	// which stand for the use of | as OR operator and not as a pipe.
+	testCases := []TestCase{
+		{
```

Comment:
```suggestion
		{
			auditCommand:          "a b c",
			expectedSlicedCommand: []string{"a b c"},
		},
		{
			auditCommand:          "a b | c",
			expectedSlicedCommand: []string{"a b", "c"},
		},
		{
			auditCommand:          "a b \\| c",
			expectedSlicedCommand: []string{"a b | c"},
		},
		{
			auditCommand:          "a b \\|\\x\\| c",
			expectedSlicedCommand: []string{"a b |\\x| c"},
		},		
		{
```

Suggesting some additional easier-to-read tests that make it a bit clearer what we are expecting